frontend/src/js/components/settings/upgrade.js

```diff
@@ -74,15 +74,14 @@ const upgradeNotes = {
 };
 
 export const Upgrade = () => {
-  const offerValid = dayjs().isBefore('2021-01-01');
   const [addOns, setAddOns] = useState([]);
   const [updatedPlan, setUpdatedPlan] = useState(PLANS.os.id);
-  const [upgraded, setUpgraded] = useState(false);
-  const navigate = useNavigate();
+  const [selectedPlan, setSelectedPlan] = useState(null);
+
   const dispatch = useDispatch();
   const features = useSelector(getFeatures);
   const org = useSelector(getOrganization);
-  const { addons: orgAddOns = [], plan: currentPlan = PLANS.os.id, trial: isTrial = true } = org;
+  const { addons: orgAddOns = [], plan: currentPlan = '', trial: isTrial = true } = org;
```

Comment:
the org should always have a plan, but if it doesn't the lowest plan should be assumed - is there a reason why you removed this for the empty string (here and elsewhere)?